sulat/translate.py

```
@@ -0,0 +1,63 @@
+                "role": "user",
+                "content": (
+                    f"Translate the following segment into {target_lang}, without additional explanation.\n\n"
+                    f"{text}"
+                ),
+            }
+        ],
+    }
+    if sampling_params:
+        payload["sampling_params"] = sampling_params
+
+    response = requests.post(api_url, headers=headers, json=payload)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)